src/integer.c

```diff
@@ -2202,6 +2202,43 @@ Obj FuncGCD_INT ( Obj self, Obj opL, Obj opR )
   return GcdInt( opL, opR );
 }
 
+Obj LcmInt ( Obj opL, Obj opR )
```

Comment:
I recently started using `clang-format` for new entries in this file (and a few older ones). Perhaps use this here, too? I.e. just do `git clang-format HEAD^ && git commit --amend -C HEAD`